Presentation.Web/Controllers/API/V1/AuthorizeController.cs

```diff
@@ -62,10 +66,21 @@ public HttpResponseMessage GetLogin()
 
         [Route("api/authorize/GetOrganizations")]
         [SwaggerResponse(HttpStatusCode.OK, Type = typeof(ApiReturnDTO<IEnumerable<OrganizationSimpleDTO>>))]
-        public HttpResponseMessage GetOrganizations()
+        public HttpResponseMessage GetOrganizations([FromUri]string orderBy = null, [FromUri]bool? orderByAsc = true)
         {
             var orgs = GetOrganizationsWithMembershipAccess();
 
+            if (!string.IsNullOrEmpty(orderBy))
+            {
+                if (!string.Equals(orderBy, "Name"))
+                    return BadRequest("Incorrect OrderBy Property name");
+
+                orderByAsc ??= true;
```

Comment:
don't change the value of the input argument

You can use this in stead:

```
orgs = orderByAsc.GetValueOrDefault(true) ? orgs.OrderBy(orderBy)
                        : orgs.OrderByDescending(orderBy);
```
The GetValueOrDefault takes care of the `null`  case and fals back to the default that you provide in that case.